src/OrchardCore.Modules/OrchardCore.Workflows/Controllers/WorkflowTypeController.cs

```diff
@@ -185,7 +189,14 @@ public async Task<IActionResult> BulkEdit(WorkflowTypeIndexOptions options, IEnu
                 switch (options.BulkAction)
                 {
                     case WorkflowTypeBulkAction.None:
-                        break;
+                        break;                        
+                    case WorkflowTypeBulkAction.Export:
+                        var archiveFileName = await ExportWorkflows(itemIds.ToArray());
+                        return new PhysicalFileResult(archiveFileName, "application/zip")
```

Comment:
Add new line before, also you could use `const` for mime type

Reply:
Is there a defined constant type that can be reused? This code is copied from ExportFileController.Execute

https://github.com/OrchardCMS/OrchardCore/blob/a2b60d5c3f3ad6bb8ed1f9dcfdc39ba9d343488d/src/OrchardCore.Modules/OrchardCore.Deployment/Controllers/ExportFileController.cs#L79

Reply:
No problem, I can refactor this later